cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -26,19 +26,584 @@ Cask Data Application Platform Release Notes
+  that actually were not running.
+  
+- `CDAP-3375 <https://issues.cask.co/browse/CDAP-3375>`__ -
+  Corrected the spelling of ZooKeeper in documentation and messages.
+  
+- `CDAP-3376 <https://issues.cask.co/browse/CDAP-3376>`__ -
+  Fixed a problem with being unable to deploy the SparkPageRank example application on a cluster.
+  
+- `CDAP-3378 <https://issues.cask.co/browse/CDAP-3378>`__ -
+  Fixed a problem in the CDAP-UI with namespace adapters not being found.
+  
+- `CDAP-3379 <https://issues.cask.co/browse/CDAP-3379>`__ -
+  Fixed a problem with unnecessary popup and confirmation dialogs on the adapter runs page
+  in the CDAP-UI.
+  
+- `CDAP-3386 <https://issues.cask.co/browse/CDAP-3386>`__ -
+  Fixed a problem with the Spark classes not being found when running a Spark program
+  through a Workflow in CDAP Distributed mode on HDP 2.2.
+  
+- `CDAP-3394 <https://issues.cask.co/browse/CDAP-3394>`__ -
+  Fixed a problem with the deployment of applications through the CDAP-UI.
+  
+- `CDAP-3399 <https://issues.cask.co/browse/CDAP-3399>`__ -
+  Fixed a problem with the SparkPageRankApp example spawning multiple containers in
+  distributed mode due to its number of services.
+  
+- `CDAP-3400 <https://issues.cask.co/browse/CDAP-3400>`__ -
+  Fixed an issue with warning messages about the notification system every time the CDAP
+  Standalone is restarted.
+  
+- `CDAP-3408 <https://issues.cask.co/browse/CDAP-3408>`__ -
+  Fixed a problem with running the CDAP Explore Service on CDH 5.[2,3].
+  
+- `CDAP-3432 <https://issues.cask.co/browse/CDAP-3432>`__ -
+  Fixed a bug where connecting with a certain namespace from the CLI would not immediately
+  display that namespace in the CLI prompt.
+  
+- `CDAP-3435 <https://issues.cask.co/browse/CDAP-3435>`__ -
+  Fixed an issue where the program status was shown as running even after it is stopped.
+  
+- `CDAP-3442 <https://issues.cask.co/browse/CDAP-3442>`__ -
+  Fixed a problem that caused application creation to fail if a config setting was given
+  to an application that does not use a config.
+  
+- `CDAP-3449 <https://issues.cask.co/browse/CDAP-3449>`__ -
+  Fixed a problem with the readless increment co-processor not handling multiple readless
+  increment columns in the same row.
+  
+- `CDAP-3452 <https://issues.cask.co/browse/CDAP-3452>`__ -
+  Fixed a problem that prevented explore service working on clusters with secure hive 0.14.
+  
+- `CDAP-3458 <https://issues.cask.co/browse/CDAP-3458>`__ -
+  Fixed a problem where streams events that had already been processed were re-processed in flows.
+  
+- `CDAP-3470 <https://issues.cask.co/browse/CDAP-3470>`__ -
+  Fixed an issue with error messages being created during a master process restart.
+  
+- `CDAP-3472 <https://issues.cask.co/browse/CDAP-3472>`__ -
+  Fixed a problem with the error message created when trying to stop a program started by a workflow.
```

Comment:
Fixed the error message returned when trying to stop a program started by a workflow.